SECURITY.md

```diff
@@ -2,12 +2,62 @@
 
 ## Reporting a Vulnerability
 
-For general cFS vulnerabilities please [open a cFS framework issue](https://github.com/nasa/cfs/issues/new/choose).
+For general cFS vulnerabilities, please [open a cFS framework issue](https://github.com/nasa/cfs/issues/new/choose). Please use the "Bug Report" template and provide as much information as possible. Apply appropraite labels for each report.  
 
-Please use the "Bug Report" template and provide as much information as possible. Apply appropraite labels for each report. For security related reports, tag the issue with the "security" label.
+For security related vulnerabilities, follow the general cFS vulnerabilites instructions and tag the issue with the "security" label.
+
+## Testing
+
+**Disclaimer: nasa/cFS is not responsible for any liability incurred under the [Apache License 2.0](https://github.com/nasa/cFS/blob/main/LICENSE).**
+
+Testing is an important aspect our team values to improve the cFS bundle. Several tools are used for testing:
+
+### CodeQL
+
+The [cFS CodeQL GitHub Actions workflow](https://github.com/nasa/cFS/actions/workflows/codeql-build.yml) is available to the public, however, the results are not available to the public. 
```

Comment:
I'd replace the results part with something like 

> fork this repository and run the CodeQL workflow to see the results

Reply:
Updated to:

>To review the results, fork the cFS repository and run the CodeQL workflow.